naemon-core.spec

```diff
@@ -4,7 +4,11 @@
 %if %{defined _with_debug}
   %define mycflags -O0 -pg -ggdb3
 %else
-  %define mycflags %{nil}
+  %if %{defined optflags}
+    %define mycflags %{optflags}
```

Comment:
I'm not 100% sure how to verify that this is actually used, but hopefully it works. If any idea how we can checked what cflags are actually used at compiletime.

Reply:
tbh, i would just remove all this mycflags. We provide standard packages. If anyone requires something specific, they have to build their own anyway.

Reply:
Ok, yeah we can do that, then the standard CFLAGS env variable should just be used I would think.